_data/toc/magento-functional-testing-framework-guide-2_3.yml

```diff
@@ -95,8 +95,15 @@ pages:
     versionless: true
 
   - label: Best practices
-    url: /mftf/2.3/best-practices.html
     versionless: true
+    children:
+
+      - label: Best practices
+        url: /mftf/2.3/best-practices.html
+        versionless: true
+      - label: Tips and Tricks
```

Comment:
lowercase tricks to follow sentence case for titles (in styleguide)

Reply:
Thanks.